src/components/Match/MatchHeader/MatchHeader.jsx

```diff
@@ -2,86 +2,89 @@ import React from 'react';
-                    .reduce(sum)
-                }
-              </Col>
-              <Col className={styles.gmde}>
-                <span className={styles.gameMode}>
-                  {transformations.game_mode(null, null, match.game_mode)}
-                </span>
-                <span className={styles.duration}>
-                  {transformations.duration(null, null, match.duration)}
-                </span>
-                <span className={styles.ended}>
-                  {strings.match_ended} {transformations.start_time(null, null, match.start_time + match.duration)}
-                </span>
-              </Col>
-              <Col className={styles.killsDire}>
-                {
-                  match.players
-                    .map(mapPlayers('kills', false))
-                    .reduce(sum)
-                }
-              </Col>
-            </Row>
-          </Col>
-          <Col xs={12} md={4} className={styles.additionalInfo}>
-            <ul>
-              <li>
-                <span>{strings.match_id}</span>
-                {match.match_id}
-              </li>
-              <li>
-                <span>{strings.match_region}</span>
-                {transformations.region(null, null, match.region)}
-              </li>
-              <li>
-                <span>{strings.match_avg_mmr}</span>
-                {(mmrPlayers.reduce(sum) / mmrPlayers.length).toFixed(0)}
-              </li>
-            </ul>
-          </Col>
-        </Row>
-      </header>
-    );
-  }
-
-  return <Spinner />;
+            </Col>
+            <Col className={styles.gmde}>
+              <span className={styles.gameMode}>
+                {transformations.game_mode(null, null, match.game_mode)}
+              </span>
+              <span className={styles.duration}>
+                {transformations.duration(null, null, match.duration)}
+              </span>
+              <span className={styles.ended}>
+                {strings.match_ended} {transformations.start_time(null, null, match.start_time + match.duration)}
+              </span>
+            </Col>
+            <Col className={styles.killsDire}>
+              {
+                match.players
+                  .map(mapPlayers('kills', false))
+                  .reduce(sum)
+              }
+            </Col>
+          </Row>
+        </Col>
+        <Col xs={12} md={4} className={styles.additionalInfo}>
+          <ul>
+            <li>
+              <span>{strings.match_id}</span>
+              {match.match_id}
+            </li>
+            <li>
+              <span>{strings.match_region}</span>
+              {transformations.region(null, null, match.region)}
+            </li>
+            <li>
+              <span>{strings.match_avg_mmr}</span>
+              {(mmrPlayers.reduce(sum) / mmrPlayers.length).toFixed(0)}
+            </li>
+          </ul>
+        </Col>
+      </Row>
+      {!match.version &&
+      <div className={styles.unparsed}>
```

Comment:
could we isolate this in a component?  We might want to reuse it for the Distributions warning.